### PR TITLE
feat: implement wizard

### PR DESCRIPTION
**Pull request by artabr**
- use openai sdk
- create different api routes

### CODE REVIEW

**Review by viktorkyssa — DISMISSED** [review]

File: src/pages/builder/index.tsx [hunk 1]
```diff
@@ -48,7 +105,7 @@ export default function BuilderPage() {
         <EditForm />
       </div>
       <div style={{ border: '1px solid red', width: '100%', height: '100%' }}>
-        <CvViewer cv={cvData} />
+        <CvViewer cv={cvDataFromWizard} />
       </div>
```
Comment (BukovY) [inline src/pages/builder/index.tsx]: pay attention that you get cv from right context

**Review by igorprotsiuk — DISMISSED** [review]

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -32,13 +34,15 @@ export default function App({ Component, pageProps }: AppProps) {
...
-          <ResumeFormProvider>
-            <CvContextProvider>
-              <MainLayout>
-                <Component {...pageProps} />
-              </MainLayout>
-            </CvContextProvider>
-          </ResumeFormProvider>
+          <ConfigProvider locale={enUs}>
+            <ResumeFormProvider>
+              <CvContextProvider>
```
Comment (igorprotsiuk) [inline src/pages/_app.tsx:39]: we should combine ResumeFormProvider and ResumeFormProvider into one

Comment (artabr) [reply src/pages/_app.tsx:39]: I believe we should

File: src/pages/builder/index.tsx (hunk 1 above)
Comment (artabr) [reply src/pages/builder/index.tsx]: I deliberately set this like this, to show the results of the Wizard

Comment (artabr) [reply src/pages/builder/index.tsx]: I used data from the Wizard as a default values for CVContext, let's leave CVContext too

**Review by BukovY — APPROVED** [review]

File: src/pages/_app.tsx (hunk 2 above)
Comment (artabr) [reply src/pages/_app.tsx:39]: We use WizardContext as a default for CVContext

**Pull request merged**